### ISSUE TITLE
Can't access certain users' profile pages.

### ISSUE THREAD

**Issue by bkw7** [djmailhot/BeatTide#78]
Description: It appears that I cannot access the profile page of the user with ID 1.

To Reproduce: On beattide.herokuapp.com try to go to Alex Miller's profile page (has ID of 1) by clicking on his name on the Top Songs list, and it will redirect to home.  Also trying to enter in any url for the user (beattide.herokuapp.com/users/2) redirects to the main page.

Expected functionality: The specific profile page should appear.

Actual functionality: Redirects to the main page.

**Comment by djmailhot** [comment]
I'v repro'd this

**Comment by djmailhot** [comment]
In the javascript (which is ridiculously AJAXy, thus hard to follow), there are a bunch of "!1"s in the code.  Could one of these be in an errornous place so to prevent access to user of id == 1?

**Comment by alexdmiller** [comment]
David, are you referring to the lines like "Path.map("#!/:1")"?

It could indeed be causing the problem... I will investigate.

On Wed, Mar 7, 2012 at 12:38 AM, djmailhot
reply@reply.github.com
wrote:

**Comment by alexdmiller** [comment]
It did have to do with those Path maps. Fixed!
